io/src/lib.rs

```diff
@@ -80,12 +92,13 @@ pub enum SupplyChainAction {
     /// and a producer of this item.
     /// * Item's [`ItemState`] must be [`PurchasedByDistributor`](ItemState::PurchasedByDistributor).
     ///
-    /// # Arguments
-    /// * `item_id`: an item's ID.
-    /// * `approve`: yes ([`true`]) or no ([`false`]).
-    ///
-    /// On success returns [`SupplyChainEvent::Success`].
-    ApproveByProducer { item_id: ItemId, approve: bool },
+    /// On success, returns [`SupplyChainEvent::Success`].
+    ApproveByProducer {
+        /// An item's ID.
+        item_id: ItemId,
+        /// Yes ([`true`]) or no ([`false`]).
```

Comment:
No need for links to bool values I guess. Example: https://doc.rust-lang.org/src/core/option.rs.html#538
```suggestion
        /// Yes (`true`) or no (`false`).
```

Reply:
Ah, somebody needs to check what does `true` or `false` mean? 😄  Ok if so.